python/tester/upload.py

```diff
@@ -112,7 +115,7 @@ def main():
         _log.error(f"No raw files found for {instrument}, aborting.")
 
 
-def get_samples(bucket, instrument):
+def get_samples_non_lsst(bucket, instrument):
     """Return any predefined raw exposures for a given instrument.
```

Comment:
Some filename conventions are still assumed but the conventions are different for LSST and non-LSST. I rephrased this and added some more docs.

Reply:
I meant that for HSC, we were free to choose whatever path was convenient for PP. We couldn't do that for LATISS because the Summit(?) team put their foot down.

Reply:
Ah, I see what you meant.  

I think the added notes has clarified that.